### DISCUSSION TITLE
Why did you remove brushes?

### DISCUSSION THREAD

**Discussion by AlmostMikhail** [friction2d/friction#377] [Q&A]
I cant find brushes in friction. Did you @rodlie perhaps move it somewhere?
![Brush](https://github.com/user-attachments/assets/fc4c26c6-af7c-4711-bc8f-60c17eb976c1)

**Comment by rodlie** [comment]
The mypaint stuff was removed before the first public release of Friction.

The state of the (paint) implementation was not something I wanted to maintain or continue develop. There are still parts of the backend left, but it's not functional and will be removed in a future release.

We could reintroduce paint support in the future. Note that I want to keep things simple and would rather make a proper API where things like this could be implemented as an extension.

Also note that the ORA stuff is also gone, as it required the libmypaint surface to function. ORA will be added back at some point, as part of a new raster library/API (that will need to be developed).

*Friction is not enve, it's a hard fork with different priorities and goals. enve was a good starting point, most (of enve) will be refactored/rewritten in the future.*
